### ISSUE TITLE
Zim-zotero plugin cannot work with non-latin-name items from Zotero collection

### ISSUE THREAD

**Issue by Nick222** [shivams/zim-zotero-plugin#1]
Zim-zotero plugin cannot work with non-latin-name items from Zotero collection :(

Can You make something to change it ?

**Comment by shivams** [comment]
@Nick222 : I will look into it. Really busy these days. Will try to take some time out to fix this.

**Comment by shivams** [comment]
@Nick222 : #7 adds support for [betterbibtex](https://retorque.re/zotero-better-bibtex/installation/). See if this resolves your issue and let me know.

**Comment by Nick222** [comment]
After install in Zotero BetterBibTeX - Zotero go in circle: Opened, BetterBibTex initializing - Need Restart Zotero - and again...

**Comment by Nick222** [comment]
Zotero 5.60 (latest version)
When BetterBibTeX disable - Citation in Zim do not work.

**Comment by shivams** [comment]
> When BetterBibTeX disable - Citation in Zim do not work.

@Nick222 Yes. You will have to change your link format preferences if you disable BetterBibTeX. By default, it is set to betterbibtex.

**Comment by Nick222** [comment]
Update comment. See above, please...

**Comment by Nick222** [comment]
May be it need to install "pandoc-zotxt.lua" - but how user can do it ?

**Comment by Nick222** [comment]
After installing some TexLive packets in OS and then installing Better Bibtex to Zotero - it appear column "Citekey" in Zotero.

Now I can insert citation in Zim by this Citerey (in Latin), but not in non-Latin.

Can I do it for citation with non-latin name now?

**Comment by Nick222** [comment]
When I click on citation in Zim - it opened browser and it say that "zotero://127.0.0.1:.." cannot find.

How I can change OS or browser or Zotero options to open such links?

**Comment by shivams** [comment]
> When I click on citation in Zim - it opened browser and it say that "zotero://127.0.0.1:.." cannot find.
> 
> How I can change OS or browser or Zotero options to open such links?

@Nick222  Is this for non-latin names or for any link? 

I think it has stopped working for me too, even for non-latin names. Give me a few days to fix this.

**Comment by Nick222** [comment]
For any link...

Thank You :+1: )

**Comment by Nick222** [comment]
In Better-Bibtex - https://github.com/retorquere/zotero-better-bibtex/issues/1487

I use this option:

[Auth]_[Title:select=1,3]_[year]
unchecking "Force citation key to plain text"

and I get CiteKey in Russian !

But I cannot insert any citation in Zim...

**Comment by Nick222** [comment]
And what is news about... ?

**Comment by Nick222** [comment]
How about adapt to Zim 0.73.* ?

**Comment by shivams** [comment]
@Nick222 Yes. I will update it for the new Zim in a few days. Thanks for the request.
